src/BookNavigator/book-navigator.js

```diff
@@ -2,8 +2,8 @@
 import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
 // eslint-disable-next-line no-unused-vars
 import { ModalManager } from '@internetarchive/modal-manager';
-import { css, html, LitElement } from 'lit-element';
-import { nothing } from 'lit-html';
+import { css, html, LitElement } from 'lit';
+import { nothing } from 'lit';
```

Comment:
Since these are now from the same package, combine them here and anywhere else there are multiple `from 'lit'` imports.

```suggestion
import { css, html, LitElement, nothing } from 'lit';
```